src/MyMind.Web/Domain/Emotions/Services/IEmotionsService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MyMind.Web.Domain.Emotions.Services
+{
+    public interface IEmotionsService
+    {
+        IQueryable<Emotion> GetAllEmotions();
```

Comment:
tu bezpieczniej zwracać IEnumarable

IQueryable to bezpośrednie operacje na bazie i nie chcemy, żeby nasz serwis to zwracał